.storybook/preview.ts

```diff
@@ -0,0 +1,37 @@
+import { Preview } from '@storybook/react';
+import { IntlProvider } from 'react-intl';
+
+import features from '../examples/src/features';
+
+import '../src/styles/variables';
+import '../src/styles/base.scss';
+
+// Constants
+global.FEATURE_FLIPS = global.FEATURE_FLIPS || features; // cannot use the word FEATURES, storybook uses it
```

Comment:
flag is the terminology we use over flip:
https://github.com/box/box-ui-elements/blob/master/src/elements/common/feature-checking/FeatureFlag.js
```suggestion
global.FEATURE_FLAGS = global.FEATURE_FLAGS || features; // cannot use the word FEATURES, storybook uses it
```